src/store/browser/fetchQueries.js

```diff
@@ -1,40 +1,12 @@
 import JSCookie from 'js-cookie';
```

Comment:
As mentioned in the description, there are now two `fetchQueries` modules - one for browser, one for server. This one is marked as a 'move' because it is most similar to the old `fetchUtils`  module.